doc/source/preparing_releases_and_hotfixes.rst

```diff
@@ -176,6 +176,13 @@ Create a release of the UserGuide
 
 For now, the UserGuide is released at the same time as the core library. To make a release of the UserGuide you should:
 
+#. Make a Pull Request with a re-generated ``releases.md`` which contains a copy of the GitHub release notes. This can be generated by doing:
+
+    .. code-block:: bash
+
+        cd doc/source/scripts
+        python gen_release_notes.py
```

Comment:
Should the new Github Python package dependency be added somewhere? (So that is included in the environment one would use to develop and release the user guide?)

Reply:
Yeah can do, I went back and forth on this one, but it does make sense to have it there as "something that's needed to run the scripts"